source/loader/layers/sanitizer/asan_interceptor.cpp

```diff
@@ -26,6 +31,11 @@ constexpr int kUsmSharedRedzoneMagic = (char)0x83;
 constexpr int kMemBufferRedzoneMagic = (char)0x84;
 constexpr int kDeviceGlobalRedZoneMagic = (char)0x85;
 
+const int kUsmDeviceDeallocatedMagic = (char)0x91;
+const int kUsmHostDeallocatedMagic = (char)0x92;
+const int kUsmSharedDeallocatedMagic = (char)0x93;
+const int kMemBufferDeallocatedMagic = (char)0x93;
+
```

Comment:
Why not using `constexpr int` like `kDeviceGlobalRedZoneMagic` above?

Reply:
done